### PR TITLE
Added api products widgets (grid and tiles)

### LINKED ISSUE #1072: Add a widget for displaying Products an API belongs to
1. **API: products**
2. **API: products (tiles)**

### PR DESCRIPTION
**Pull request by ygrik**
closes #1072

### CODE REVIEW

File: src/components/apis/api-products/apiProductsContract.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { Contract } from "@paperbits/common";
+import { HyperlinkContract } from "@paperbits/common/editing";
+
+/**
+ * Product API list widget configuration.
+ */
+export interface ApiProductsContract extends Contract {
+    /**
+     * List layout.
```
Comment (azaslonov) [inline src/components/apis/api-products/apiProductsContract.ts:9]: Please add value examples to the comment.

File: src/components/apis/api-products/apiProductsModel.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { HyperlinkModel } from "@paperbits/common/permalinks";
+
+/**
+ * Product API list widget configuration.
+ */
+export class ApiProductsModel {
+    /**
+     * List layout.
```
Comment (azaslonov) [inline src/components/apis/api-products/apiProductsModel.ts:8]: Also, examples please.

**Review by azaslonov — APPROVED** [review]

**Comment by AnRei123** [comment]
Currently I still cannot see the new widget "API products (grid and tiles)". In the widget selection window, I currently can only see the widget "Product: API". Does it take time until this change is reflected for the EU region? I wondered as the change with the list of operations that is part of the same release is already visible in my EU-related environment.

**Comment by mikebudzynski** [comment]
@AnRei123 I can see the widget in my upgraded service. The widget is in the "Products" category. Can you please check again in a few days?

**Pull request merged**